scripts/ci-prepare-contour.sh

```diff
@@ -1,19 +1,20 @@
 #! /bin/bash
 
-set -e
+set -ex
+
+BUILD_DIR=${BUILD_DIR:-build}
+CMAKE_BUILD_TYPE="${CMAKE_BUILD_TYPE:-RelWithDebInfo}"
 
 prepare_build_ubuntu()
 {
-   mkdir -pv ${BUILD_DIR}
-   cd ${BUILD_DIR}
-   cmake .. \
+   cmake \
       -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} \
-      -DCMAKE_CXX_COMPILER="${CXX}" \
```

Comment:
Where did this line go, and why did it disappear?

Reply:
I removed it in order to get cmake happy finding the compiler. TBH, I'm not sure we need or do not need this line.